clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7732,6 +7731,13 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
 
   addOpenMPHostOffloadingArgs(C, JA, Args, CmdArgs);
 
+  // Temporarily disable this for LTO if it's not explicitly enabled.
```

Comment:
How is this being disabled for LTO?

Reply:
Oh I think it is happening in the LLVM side? If that's what you are referring to you should probably clarify that here (e.g. that on the LLVM side this is being temporarily ignored)